apps/site/src/app/admin/applicants/[uid]/components/ApplicationReviews.tsx

```diff
@@ -0,0 +1,43 @@
+import { useContext } from "react";
+
+import ApplicantStatus from "@/app/admin/applicants/components/ApplicantStatus";
+import { Review, uid } from "@/lib/admin/useApplicant";
+import UserContext from "@/lib/admin/UserContext";
+
+interface ApplicationReviewsProps {
+	reviews: Review[];
+}
+
+function ApplicationReviews({ reviews }: ApplicationReviewsProps) {
+	const { uid } = useContext(UserContext);
+
+	if (reviews.length === 0) {
+		return <p>-</p>;
+	}
+
+	const formatUid = (uid: uid) => uid.split(".").at(-1);
```

Comment:
Nitpick: The `uid` variable and the `uid` type may be a bit confusing, at least to me. Should there be some sort distinction made between the two (perhaps in name)?

Reply:
Convention is type names are usually capitalized, not sure why I didn't do that before.